kuma/payments/tests/test_views.py

```diff
@@ -38,10 +41,117 @@ def test_error_view(mock_enabled, client, settings):
 @pytest.mark.django_db
 @pytest.mark.parametrize('view', ('payments',
                                   'payment_succeeded',
-                                  'payment_error'))
+                                  'payment_error',
+                                  'recurring_payment_initial',
+                                  'recurring_payment_subscription'))
 @mock.patch('kuma.payments.views.enabled')
 def test_views_404(mock_enabled, view, client, settings):
     """If disabled, contribution pages are 404."""
     mock_enabled.return_value = False
     response = client.get(reverse(view))
     assert response.status_code == 404
+
+
+class RecurringPaymentTests(UserTestCase):
```

Comment:
Nit: We avoid ``UserTestCase`` for new tests, and test classes in general. I believe it is unnecessary here, and these could be converted to functions like ``test_contribute_view``.

Reply:
I have changed tests to not use UserTestCase